core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -364,12 +364,12 @@ public void draw(float delta) {
         level.draw(canvas, delta, debugFont);
 
         // Final message
-        if (isSuccess) {
+        if (!isFailed && isSuccess) {
             displayFont.setColor(Color.YELLOW);
             canvas.begin(); // DO NOT SCALE
             canvas.drawText("VICTORY!", displayFont, 0, canvas.getHeight());
             canvas.end();
-        } else if (isFailed) {
+        } else if (!isSuccess && isFailed) {
```

Comment:
Doesn't this just mean nothing is displayed if the enemy runs into the player after winning?

Reply:
Yep. Tbh this pull request was done last night in the hopes that it would be merged in before today, it wasn't a permanent fix.